assets/user-profile/reducers.js

```diff
@@ -44,6 +45,16 @@ export default function itemReducer(state = initialState, action) {
             user: action.user,
         };
     }
+
+    case EDIT_USER: {
```

Comment:
I'm not sure if field changes should make it to the store, imo it's  more like internal state of the user profile component